src/app/tests/suites/certification/Test_TC_MOD_1_1.yaml

```diff
@@ -77,27 +99,58 @@ tests:
               contains: [4]
 
     - label:
-          "TH reads the optional attribute(OnMode) in AttributeList from the DUT"
-      PICS: MOD.S.A0005
+          "Read the Feature dependent(MOD.S.F00 - DEPONOFF) and optional attribute(OnMode) in AttributeList from the DUT"
+      PICS: MOD.S.A0005 && MOD.S.F00
```

Comment:
Shouldn't this drop the attribute PICs bit now?  If MOD.S.F00 the attribute is mandatory.

Or is something else checking that the PICs match the conformance requirements here?

Reply:
@cjandhyala @raju-apple

Reply:
@bzbarsky-apple 

The test plan says this: 

"The list include entries based on feature support:
0x0005 SHALL be included if and only if MOD.S.F00(DEPONOFF)
and SHALL NOT be included otherwise."

So I wonder if this should be updated as follows:

a) At this step, if MOD.S.F00: check that the return value contains this attribute id (5)
b) Add another step: if !MOD.S.F00 check that the return value excludes this attribute id (5)